components/org.wso2.carbon.identity.application.authenticator.biometric/src/main/java/org/wso2/carbon/identity/application/authenticator/biometric/dao/impl/DeviceDAOImpl.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ *
+ */
+
+package org.wso2.carbon.identity.application.authenticator.biometric.dao.impl;
+
+import org.wso2.carbon.identity.application.authenticator.biometric.dao.DeviceDAO;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * performs DAO operations related to Biometric device store which stores the device IDs against the usernames.
+ */
+public class DeviceDAOImpl implements DeviceDAO {
+
+    private static DeviceDAOImpl deviceDAO = new DeviceDAOImpl();
+    private Map<String, List<String>> deviceIDStore = new HashMap<>();
+
+    private DeviceDAOImpl() {
+
+        List<String> dewniDevices = new ArrayList<>();;
+        dewniDevices.add("ca0OWsg-30c:APA91bHQj3LRdOt7BgQJsOGbz_uWBFe8BtBLS0WSXZFNnVj6BZkU7PM__" +
```

Comment:
Are these dummy values? Unless we need to remove them.

Reply:
done